sentry-core/src/performance.rs

```diff
@@ -168,6 +167,22 @@ impl TransactionOrSpan {
         }
     }
 
+    /// Set the status of the Transaction/Span.
```

Comment:
I'd really like it if there where pointers to the docs on what this status is and what setting it does.